src/docs-website/docs/api/for-cities/historical-data.md

```
@@ -0,0 +1,202 @@
+
+---
+
+## Overview
+
+To query historical data for your Grid, use the site names or device names from your Grid in the Analytics API request. You can discover the sites in your Grid by calling the [recent measurements endpoint](./recent-measurements) first and collecting the `site_id` and `device` values.
+
+---
+
+## Endpoint
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)